services/notify/src/notifier.js

```diff
@@ -157,6 +158,27 @@ class Notifier {
     this.markSent(roomId, format, formattedBody, body, msgtype);
     this.monitor.log.matrix({ dest: roomId });
   }
+
+  async slack({ channelId, text, blocks, attachments }) {
+    if (!this._slack) {
+      debug('Slack is not configured.');
```

Comment:
maybe we could use a `this.monitor.warning` here just to make answering "why didn't I get my slack messages" questions easier in the future

Reply:
That's a good idea, this did seem like a bit of a weak logging mechanism when I wrote it. I'll sort that out.